src/hec/firo/CsvEnsembleReader.java

```diff
@@ -0,0 +1,140 @@
+package hec.firo;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+
+public class CsvEnsembleReader {
+
+
+    String path; // path to csv files
+
+    public CsvEnsembleReader(String path) {
+        this.path = path;
+    }
+
+    // Allegedly threadsafe
+    // https://docs.microsoft.com/en-us/dotnet/api/system.console?redirectedfrom=MSDN&view=netframework-4.8
+    static boolean DebugMode = false;
+
+    static void Log(String msg) {
+        if (DebugMode)
+            System.out.println(msg);
+    }
+
+    static void LogWarning(String msg) {
```

Comment:
CWMS ATO disallows writing to system out. we should consider leveraging a messaging/logging system. I wrote one called [core](https://github.com/HenryGeorgist/Core), but there is an implementation in the shared code for another - we should consider either.

Reply:
I have added a Simple Logger class, with a couple static methods -- This will avoid any additional dependencies for now, and a single point to manage.